sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -646,6 +650,17 @@ which is used in a graph will be kept alive on the host for the lifetime of the
 graph. Destroying that buffer during the lifetime of a `command_graph`
 constructed with this property results in undefined behavior.
 
+===== Enable-Profiling Property [[enable-profiling]]
+
+The `property::graph::enable_profiling` property enable profiling
+the executable_graph.
+This property is passed to the `command_graph::finalize()` function.
+If this property is not passed to the finalize function, certain optimizations,
+which prevent graph profiling, may be applied to this executable graph.
+In this case, an error will be thrown when attempting to profiling an event from 
+a graph submission that was created without this property.
+Note that this limitation currently only applies to the zero-level backend. 
```

Comment:
I think we should remove this line about level zero, and all backends should throw if this property isn't set for more consistent behaviour.